CHANGELOG.md

```diff
@@ -16,7 +16,9 @@ avoid adding features or APIs which do not map onto the
 
 ## Unreleased
 
-- None
+## [4.0.0b6] - 2024-08-23
```

Comment:
Let's just put this under `## Unreleased` for now, since we'll publish as a separate step.

Reply:
OK!